doc/source/index.rst

```diff
@@ -329,6 +329,32 @@ Options:
   - ``min_valid_data_fraction: 10`` - only generate products if at least 10% of covered
   part of scene contains valid data.
 
+Uploading produced data to S3
+*****************************
+
+.. note::
+   In most cases it is perhaps better to have a separate dispatcher from
+   `Trollmoves <https://github.com/pytroll/trollmoves/>`_ to handle the file transfers. With
+   containers this will require an additional container image, configuration and a use of
+   a shared volume for the temporary local files, so this plugin makes setting up the uploading
+   easier. Later on, if direct saving to S3 becomes available in Satpy writers, this plugin
+   will be removed.
```

Comment:
I still don't agree, sorry. I propose we skip the container part and explain what this is: a workaround. Here is a suggestion.

```suggestion
.. note::
   To transfer files to remote file systems, it is recommended to use a separate process, eg the dispatcher 
   from `Trollmoves <https://github.com/pytroll/trollmoves/>`_ to handle the file transfers. However,
   this plugin is a workaround for Satpy's current inability to perform direct remote writing, not a 
   dispatcher in itself. Later on, if direct saving to S3 becomes available in Satpy writers, this plugin
   will be deprecated and eventually removed.
```